includes/class-wc-calypso-bridge-setup.php

```diff
@@ -166,6 +166,25 @@ public function is_theme_installed( $theme ) {
 	public function register_payments_welcome_page() {
 		global $menu;
 
+		// WooCommerce must be active.
+		if ( !is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
+			return;
+		}
+
+		// WC Payment must not be installed.
```

Comment:
Should we just check if the plugin is active instead? We can ask for another chance by rendering this menu to those who installed the plugin then deactivated it.